// random-jokes-api/index.js
module.exports = {
    joke: function() {
        let jokes  = require('./Data/joke_list').jokes;
        let j = jokes[Math.floor(Math.random() * jokes.length)]
        return j.joke
    },
    pun: function() {
        let pun  = require('./Data/pun_list').pun;
        let p = pun[Math.floor(Math.random() * pun.length)]
        return p.joke
    },
    antijoke: function() {
        let anti  = require('./Data/anti_jokes').anti;
        let a = anti[Math.floor(Math.random() * anti.length)]
        return a.joke
    },
    quotes: function() {
        let quotes  = require('./Data/quotes_list').quotes;
        let q = quotes[Math.floor(Math.random() * quotes.length)]
        return q.quote
    },
    roast: function() {
        let roast  = require('./Data/roast_list').roast;
        let r = roast[Math.floor(Math.random() * roast.length)]
        return r.roasts
    },
    uselessweb: function() {
        let web  = require('./Data/uselesswebList').web;
        let w = web[Math.floor(Math.random() * web.length)]
        return w.url
    },
    showerThought: function() {
        let shower  = require('./Data/shower_list').shower;
        let s  = shower[Math.floor(Math.random() * shower.length)]
        return s.thought
    },
    meme: function() {
        let memes  = require('./Data/memes_list.json');
        let m  = memes[Math.floor(Math.random() * memes.length)]
        return m
    },
    chuckNorris: function() {
        let chunk  = require('./Data/chunk_norris.json');
        let c  = chunk[Math.floor(Math.random() * chunk.length)]
        return c.norris
    },
    trivia: function() {
        let trivia  = require('./Data/trivia_q.json');
        let t  = trivia[Math.floor(Math.random() * trivia.length)]
        return t
    },
    copmliment: function() {
        let copmliment  = require('./Data/compliment.json');
        let c  = copmliment[Math.floor(Math.random() * copmliment.length)]
        return c.here
    },
    truth: function() {
        let truth  = require('./Data/truth.json');
        let t  = truth[Math.floor(Math.random() * truth.length)]
        return t.truth
    },
    dare: function() {
        let dare  = require('./Data/dare.json');
        let d  = dare[Math.floor(Math.random() * dare.length)]
        return d.dare
    },
    wouldYouRather: function() {
        let would  = require('./Data/Wouldyourather.json');
        let w  = would[Math.floor(Math.random() * would.length)]
        return w.here
    },
    neverHaveIEver: function() {
        let would  = require('./Data/neverhaveiever.json');
        let w  = would[Math.floor(Math.random() * would.length)]
        return w.here
    },
    cat: function() {
        let cat  = require('./Data/cat.json');
        let c  = cat[Math.floor(Math.random() * cat.length)]
        return c.url
    },
    dog: function() {
        let dog  = require('./Data/dog.json');
        let d  = dog[Math.floor(Math.random() * dog.length)]
        return d.url
    },
    snake: function() {
        let snake  = require('./Data/snake.json');
        let s  = snake[Math.floor(Math.random() * snake.length)]
        return s.url
    },
    fox: function() {
        let fox  = require('./Data/fox.json');
        let f  = fox[Math.floor(Math.random() * fox.length)]
        return f.url
    },
    redpanda: function() {
        let redpanda  = require('./Data/redpanda.json');
        let r  = redpanda[Math.floor(Math.random() * redpanda.length)]
        return r.url
    },
    lizard: function() {
        let lizard  = require('./Data/lizard.json');
        let l  = lizard[Math.floor(Math.random() * lizard.length)]
        return l.url
    },
    otter: function() {
        let otter  = require('./Data/otter.json');
        let o  = otter[Math.floor(Math.random() * otter.length)]
        return o.url
    },
    bunny: function() {
        let bunny  = require('./Data/bunny.json');
        let b  = bunny[Math.floor(Math.random() * bunny.length)]
        return b.url
    }
};
